### ISSUE TITLE
Accuracy Test  v0.9.3  in fails to collect data

### ISSUE THREAD

**Issue by IngaIbs** [pupil-labs/pupil#682]
Hi!
We are trying to perform the accuracy test but get the following error after the performance of the test:

```
world - [INFO] calibration_routines.accuracy_test: Stopping Accuracy_Test
world - [INFO] calibration_routines.accuracy_test: Collected 0 data points.
world - [WARNING] calibration_routines.accuracy_test: Did not collect enough data.
```

We are running the newest version (v0.9.3) from source with python 3.4.

**Comment by papr** [comment]
Dear Inga,
the accuracy test only collects data points which have a high confidence. ([source](https://github.com/pupil-labs/pupil/blob/master/pupil_src/shared_modules/calibration_routines/accuracy_test.py#L147))

You can see the current detection confidences in their respective graphs in the `world` window.

**Comment by behinger** [comment]
I can confirm this bug in v0.9.2 in ubuntu 16.04 on two other machines with different eyetrackers.
From source or from binary. Confidence was above .6. I think the calibration procedure would not calibrate if confidence wouldn't be above .6.

Also the accuracy test would not finish if it wouldn't detect the eye (not 100% sure about this one)

[see also google-group here](https://groups.google.com/forum/#!searchin/pupil-discuss/obioma%7Csort:relevance/pupil-discuss/_7KBehNE_DA/Vl0_zOcGBQAJ)

Can you confirm this on your own setup?

**Comment by papr** [comment]
Yes, we were able to replicate the issue. The problem was that the accuracy test plugin used the deprecated `update()` method instead of `recent_events()`. Got fixed with d19554f. Thanks for pointing out the bug. This should have been caught by out test procedure. :man_facepalming:
